assemble/conf/log4j2-service.properties

```diff
@@ -40,30 +40,30 @@ appender.console.filter.threshold.level = info
 appender.rolling.type = RollingFile
 appender.rolling.name = LogFiles
 appender.rolling.fileName = ${filename}.log
-appender.rolling.filePattern = ${filename}-%d{MM-dd-yy-HH}-%i.log.gz
+appender.rolling.filePattern = ${filename}-%d{yyyy-MM-dd}-%i.log.gz
 appender.rolling.layout.type = PatternLayout
 appender.rolling.layout.pattern = %d{ISO8601} [%-8c{2}] %-5p: %m%n
 appender.rolling.policies.type = Policies
 appender.rolling.policies.time.type = TimeBasedTriggeringPolicy
 appender.rolling.policies.time.interval = 1
 appender.rolling.policies.time.modulate = true
 appender.rolling.policies.size.type = SizeBasedTriggeringPolicy
-appender.rolling.policies.size.size=100MB
+appender.rolling.policies.size.size=512MB
 appender.rolling.strategy.type = DefaultRolloverStrategy
 appender.rolling.strategy.max = 10
 
 appender.audit.type = RollingFile
 appender.audit.name = AuditLogFiles
 appender.audit.fileName = ${filename}.audit
-appender.audit.filePattern = ${filename}-%d{MM-dd-yy-HH}-%i.audit.gz
+appender.audit.filePattern = ${filename}-%d{yyyy-MM-dd}-%i.audit.gz
 appender.audit.layout.type = PatternLayout
 appender.audit.layout.pattern = %d{ISO8601} [%-8c{2}] %-5p: %m%n
 appender.audit.policies.type = Policies
 appender.audit.policies.time.type = TimeBasedTriggeringPolicy
 appender.audit.policies.time.interval = 1
 appender.audit.policies.time.modulate = true
 appender.audit.policies.size.type = SizeBasedTriggeringPolicy
-appender.audit.policies.size.size=100MB
+appender.audit.policies.size.size=512MB
 appender.audit.strategy.type = DefaultRolloverStrategy
 appender.audit.strategy.max = 10
```

Comment:
The max files could be decreased to keep relative usage the same.  The files compress well, so maybe 5?

Reply:
I don't think it's necessary. 10 is still modest enough.